Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -511,21 +538,20 @@ - (NSMutableDictionary *)adOptionsForServerParameters:(NSDictionary<NSString *,
     if ( [serverParameters al_containsValueForKey: @"is_muted"] ) // Introduced in 6.10.0
     {
         BOOL muted = [serverParameters al_numberForKey: @"is_muted"].boolValue;
-        [VungleSDK sharedSDK].muted = muted;
-        
-        options[VunglePlayAdOptionKeyStartMuted] = @(muted);
+        // Overwritten by `mute_state` setting for Vungle only if YES,
```

Comment:
Agree with @stanley-vungle - this line should be updated:

`The mute status for Vungle will only be overwritten if YES.`